### PR TITLE
Add hotkeys, incl. macOS + Windows global hotkeys

### PR DESCRIPTION
**Pull request by karlhorky**
Updated hotkeys taken from this PR: https://github.com/flameshot-org/flameshot/pull/1097

Ref: https://github.com/flameshot-org/flameshot/issues/1593

### CODE REVIEW

File: docs/guide/key-bindings.md [hunk 1]
```diff
@@ -3,21 +3,41 @@
...
+| <kbd>Shift</kbd> + <kbd>Print</kbd>                                            | Screenshot History                                     |
 
 <kbd>Shift</kbd> + drag a handler of the selection area: mirror redimension in the opposite handler.
 
 ### Global
-If you want use Flameshot as a default screenshot utility, chances are you want to launch it using the <kbd>Prt Sc</kbd> key. Flameshot doesn't yet offer a fully-automated option to do so, but you can configure your system to do so.
+
+Flameshot uses <kbd>Print screen</kbd> (Windows) and <kbd>cmd</kbd>-<kbd>option</kbd>-<kbd>shift</kbd>-<kbd>4</kbd> (macOS) as default global hotkeys.
+
+On Linux, Flameshot doesn't yet support <kbd>Pr Scr</kbd> out of the box, but with a bit of configuration you can set this up:
```
Comment (mmahmoudian) [inline docs/guide/key-bindings.md:40]: These are already in the other PR (#1845)

File: docs/guide/key-bindings.md [hunk 2]
```diff
@@ -3,21 +3,41 @@
...
+| <kbd>Ctrl</kbd> + <kbd>Q</kbd>                                            | Leave the capture screen                                         |
+| <kbd>Ctrl</kbd> + <kbd>O</kbd>                                            | Choose an app to open the capture                                |
+| <kbd>Return</kbd>                                             | Upload the selection to Imgur                                      |
+| <kbd>Spacebar</kbd>                                                       | Toggle visibility of sidebar with options of the selected tool, color picker for the drawing color and history menu |
+| Right Click                                                               | Show the color wheel                                              |
+| Mouse Wheel                                                               | Change the tool's thickness                                    |
+| <kbd>Print screen</kbd>                                          | Capture Screen |
+| <kbd>Shift</kbd> + <kbd>Print</kbd>                                            | Screenshot History                                     |
```
Comment (mmahmoudian) [inline docs/guide/key-bindings.md:32]: Considering that these, unlike others, are not for the GUI mode, and considering that they are not universal (e.g Mac does not even have <kbd>PrntScrn</kbd> button), I suggest removing these.

File: docs/guide/key-bindings.md (hunk 1 above)
Comment (karlhorky) [reply docs/guide/key-bindings.md:40]: Replied over there: https://github.com/flameshot-org/flameshot/pull/1845#issuecomment-907657878

File: docs/guide/key-bindings.md (hunk 2 above)
Comment (karlhorky) [reply docs/guide/key-bindings.md:32]: These are changes that I took from @DbDibyendu's PR flameshot-org/flameshot#1097 (merged by @borgmanJeremy), so if there should be any changes done here, then they should probably also be done to the readme in https://github.com/flameshot-org/flameshot

**Comment by ZetaoYang** [comment]
@karlhorky Can you solve the conflicts?

**Comment by karlhorky** [comment]
@ZetaoYang opened https://github.com/flameshot-org/flameshot-org.github.io/pull/86 for this, closing this PR.

**Pull request closed without merging**